audiofile/core/utils.py

```diff
@@ -7,6 +7,26 @@
 import audeer
 
 
+def broken_file_error(file: str) -> str:
+    r"""Broken file error message.
+
+    If we encounter a broken file,
+    we raise the same error for non SND files
+    as soundfile does for SND files
+
+    Args:
+        file: file name
+
+    Returns:
+        error message
+
+    """
+    return (
+        f'Error opening {file}: '
+        'File contains data in an unknown format.'
```

Comment:
The only real question I have about this PR is on why you modify the error message and raise a different built-in exception class with a custom message?

Could one not also subclass as "class BrokenFileError(Exception):" from exception and then re-raise as needed. 
Probably something like this would work:

```
try:
    frobnicate()
except subprocess.CalledProcessError as exc:
    raise BrokenFileError(.....) from exc
```
Note that the way you implement it is totally fine, I am just not seing whether there is a particular reason.

Reply:
I wanted to make sure that we always raise the same error.

If you hand in a broken WAV file `soundfile` is handling it under the hood and will raise a `RuntimeError` with the error message I reuse. For non WAV files we and up with the `subprocess.CalledProcessError` which I catch and then raise a `RuntimeError` with a similar error message as for WAV files. Does this make sense?

But I have to admit I also do not understand what `from exc` achieves.

Reply:
Thanks for the explanation, I also think that the `from None` trick can be useful in some situations.

Reply:
OK, I will stay here with the current solution.